process/smartContract/builtInFunctions/factory.go

```diff
@@ -14,120 +14,177 @@ var log = logger.GetOrCreate("process/smartContract/builtInFunctions")
 
 // ArgsCreateBuiltInFunctionContainer -
 type ArgsCreateBuiltInFunctionContainer struct {
-	GasMap               map[string]map[string]uint64
+	GasSchedule          core.GasScheduleNotifier
 	MapDNSAddresses      map[string]struct{}
 	EnableUserNameChange bool
 	Marshalizer          marshal.Marshalizer
 	Accounts             state.AccountsAdapter
 }
 
-// CreateBuiltInFunctionContainer will create the list of built-in functions
-func CreateBuiltInFunctionContainer(args ArgsCreateBuiltInFunctionContainer) (process.BuiltInFunctionContainer, error) {
-	gasConfig, err := createGasConfig(args.GasMap)
+type builtInFuncFactory struct {
```

Comment:
Maybe increase the code coverage for this?

Reply:
the factory is tested for most of the cases. it will error in case of bad arguments and it is tested.